src/dmd/storage_class.d

```diff
@@ -0,0 +1,86 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1999-2019 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/storage_class.d, _storage_class.d)
+ * Documentation:  https://dlang.org/phobos/dmd_storage_class.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/storage_class.d
+ */
+
+module dmd.storage_class;
+
+/**
+ * Enumerates the possible storage classes.
+ *
+ * Storage classes usually indicates how variables and parameters are stored, passed, copied, etc.
+ * but much more in D.
+ */
+enum StorageClass : ulong
+{
+    undefined_          = 0L,
```

Comment:
The trailing `_` is only for when the name is also a keyword. This is not the case here.